lib/__tests__/dependencyManagement.test.ts

```diff
@@ -8,21 +7,21 @@ jest.mock('fs', () => ({
   existsSync: jest.fn().mockReturnValue(true),
 }));
```

Comment:
if you add `jest.mock('util')` here you can import `{promisify}` from util and then simplify all of the promisify overrides to:

```ts
(promisify as unknown as jest.Mock).mockReturnValue(execMock);
```

There might be an even better way to do it, but that's the cleanest option I could find from playing around with it.

Reply:
Addressed in https://github.com/HubSpot/hubspot-cli/pull/1345/commits/ee15a6b4d7c9cd9b947f871abaf6af859d949b98